src/main/python/systemds/matrix/operation_node.py

```diff
@@ -14,23 +14,21 @@
 #  limitations under the License.
 # ------------------------------------------------------------------------------
 
-from __future__ import annotations
```

Comment:
this was the line / import that broke 3.6 for me.

so now it works with 3.6?

Reply:
Yes :+1: